tests/test_qnode.py

```diff
@@ -18,7 +18,7 @@
 from scipy.sparse import coo_matrix
 
 import pennylane as qml
-from pennylane import numpy as pnp
+from pennylane import QuantumFunctionError, numpy as pnp

```

Comment:
```suggestion
from pennylane import numpy as pnp
```

I think my IDE automatically made that... oops